py/desisim/test/test_quickbrick.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import, division, print_function
+
+import unittest
+from uuid import uuid1
+from shutil import rmtree
+import os
+
+import numpy as np
+from astropy.io import fits
+from desisim.scripts import quickbrick
+import desispec.io
+
+class TestQuickBrick(unittest.TestCase):
+    
+    #- Pick random test directory
+    def setUp(self):
+        self.testdir = 'test-{uuid}'.format(uuid=uuid1())
```

Comment:
It might be cleaner and more portable to use the [tempfile module](https://docs.python.org/2/library/tempfile.html) here.